manuals/1.0/en/database.media.md

```
@@ -327,6 +327,93 @@ public function testAdd(): void
+
+```php
+use Ray\MediaQuery\MediaQuerySqlTemplateModule;
+
+protected function configure(): void
+{
+    $this->install(new MediaQuerySqlTemplateModule("-- App: {{ id }}.sql\n{{ sql }}"));
+}
+```
+
+Available template variables:
+- `{% raw %}{{ id }}{% endraw %}`: Query ID
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)